### PR TITLE
update exchange pairs to include common swap symbol

### PR DESCRIPTION
**Pull request by BigslimVdub**
Prior we used aeon/btc or aeon->btc, then started adding aeon<=>any. This commit is to just centralize the symbol used so they all look the same. For example: Using Aeon=>BTC for an Aeon to BTC only transaction allowed, using BTC=>Aeon when a BTC to Aeon only transaction allowed, or when multiple exchange currencies allowed using Aeon<=>Any. 

Also adding in AeonAPI for our Aeon to BTC payment processor.

### CODE REVIEW

**Comment by BigslimVdub** [comment]
exchange style

Screenshot from website of current style of exchange symbols.

File: _layouts/default.html [hunk 1]
```diff
@@ -112,16 +112,17 @@ <h3>Subscribe to announcements</h3>
...
+              <li><a href="https://hitbtc.com/exchange/AEON-to-BTC" rel="nofollow">HitBTC</a> [aeon<=>btc]</li>
+              <li><a href="https://tradeogre.com/exchange/BTC-AEON" rel="nofollow">TradeOgre</a> [aeon<=>btc]</li>
+              <li><a href="https://bitliber.com/markets/aeon-btc" rel="nofollow">Bitliber</a> [aeon<=>btc]</li>
+              <li><a href="https://bitliber.com/markets/aeon-xmr" rel="nofollow">Bitliber</a> [aeon<=>xmr]</li>
+              <li><a href="https://letsdocoinz.com/AEON-BTC" rel="nofollow">LetsDoCoinz</a> [aeon<=>btc]</li>
+              <li><a href="https://changenow.io/?from=btc&to=aeon&amount=0.01" rel="nofollow">ChangeNOW</a> [aeon<=>any]</li>
+              <li><a href="https://simpleswap.io" rel="nofollow">SimpleSwap</a> [aeon<=>any]</li>
+              <li><a href="https://Xchange.me" rel="nofollow">Xchange</a> [any=>aeon]</li>
               <li><a href="https://bisq.network/markets/?currency=aeon_btc" rel="nofollow">Bisq</a> [aeon<=>btc]</li>
+              <li><a href="https://aeonapi.com" rel="nofollow">AeonAPI</a> [Aeon=>BTC]</li>
```
Comment (stoffu) [inline _layouts/default.html]: Maybe uncapitalize the currency symbols like the rest of the lines?

Comment (BigslimVdub) [reply _layouts/default.html]: will you squash commits if I make another commit?

**Comment by shigutso** [comment]
looks good

File: _layouts/default.html (hunk 1 above)
Comment (stoffu) [reply _layouts/default.html]: No, squashing needs to be done by you because you’re the author of this PR. You can easily do it by `git rebase -i HEAD^^`

**Comment by stoffu** [comment]
Please squash

**Comment by BigslimVdub** [comment]
I think I did that right. Check it out and let me know.

**Comment by BigslimVdub** [comment]
What a mess. I think I got it this time heh

**Comment by stoffu** [comment]
I don't mean to bug you too much, but please remove these unnecessary lines from the commit message:

> Squash commits to one
>
> Squash commits again

**Comment by BigslimVdub** [comment]
?

**Comment by BigslimVdub** [comment]
Closing to prevent issues with #69. Will modify new files as needed after a decision is made on that website overhaul. This PR was a mess anyway.

**Comment by aeon1234** [comment]
I would prefer using symbols like ⇐ , ⇔ , ⇒.
what do you think?

**Pull request closed without merging**